### DISCUSSION TITLE
Is there an API to draw the border of a triangle?

### DISCUSSION THREAD

**Discussion by 00Her0** [nesbox/TIC-80#1397] [Q&A] [answered]
Is there an API to draw the border of a triangle? Other similar functions like `rect()` and `circ()` have an API to draw just their border (`rectb()` and `circb()`). If we don't have `trib()` or something like that it might be nice to have.

**Comment by joshgoebel** [comment, marked as answer]
No, but you have `line`, so a triangle function would only be 3 lines of code.

**Reply by 00Her0** [reply]
Good point.
